packages/template-retail-react-app/app/components/order-summary/index.jsx

```diff
@@ -111,14 +111,14 @@ const OrderSummary = ({
     isEstimate = false,
     fontSize = 'md'
 }) => {
-    const {removePromoCode, ...promoCodeProps} = usePromoCode()
-    const shippingItem = basket.shippingItems?.[0]
-    const hasShippingPromos = shippingItem?.priceAdjustments?.length > 0
-
     if (!basket?.basketId && !basket?.orderNo) {
         return null
```

Comment:
Rearranging the code, so that we exit earlier → to fix a bug I encountered.

Reply:
Ah right, I forgot about that.. thanks.

Maybe we should have an eslint plugin that catches this kind of thing.

Reply:
There is no hooks come after this condition or do I miss anything? This is for OrderSummary to render null, not hook

Reply:
@alexvuong I've already fixed the problem. That's why it looks ok now :)